packages/terra-time-input/src/terra-dev-site/test/time-input/time-input/Second.test.jsx

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
+import classNames from 'classnames/bind';
 import TimeInput from '../../../../TimeInput';
+import styles from './time-input.test.module.scss';
+
+const cx = classNames.bind(styles);
 
 const TimeInputDefault = () => (
-  <div style={{ caretColor: 'transparent' }}>
+  <div className={cx(['content-wrapper'])}>
```

Comment:
These changes here and in the next two files are being taken care of in #812.

Reply:
will revert these changes here